includes/blocks/container/frontend.css.php

```diff
@@ -131,8 +131,8 @@
 }
 
 $background_video_opacity_value = ( isset( $attr['overlayOpacity'] ) && 'none' !== $attr['overlayType'] && ( ( 'color' === $attr['overlayType'] && ! empty( $attr['backgroundVideoColor'] ) ) || ( 'gradient' === $attr['overlayType'] && ! empty( $attr['gradientValue'] ) ) ) ) ? 1 - $attr['overlayOpacity'] : 1;
-
-$selectors = array(
+$bg_video_image_fallback        = isset( $attr['backgroundVideoFallbackImage'] ) && isset( $attr['backgroundVideoFallbackImage']['url'] ) ? $attr['backgroundVideoFallbackImage']['url'] : '';
```

Comment:
Would this be better?

```suggestion
$bg_video_image_fallback        = ! empty( $attr['backgroundVideoFallbackImage']['url'] ) ? $attr['backgroundVideoFallbackImage']['url'] : '';
```

Reply:
Resolved